test/testdata/reftest/reftest_wait_0.html

```diff
@@ -1,11 +1,13 @@
 <title>rel=match that should fail</title>
+<html class="reftest-wait">
```

Comment:
Should be before the title, given it's the root element. (This works, but is a needless parse error for what we're testing.)